AirSkyBoat/AirSkyBoat
# Fixes Panzer Percival to not cast spells

**_I affirm:_**


## Please enter a player-facing description

Fixes Panzer Percival to not cast spells

## What does this pull request do? (Please be technical)

Removes the spell list from Panzer_Percival in mob_pools.sql

## Steps to test these changes

Apply DB update
Bounce Server
Spawn Panzer
Fight Panzer
Note that he no longer casts spells as if he was a casting dark knight.

## Special Deployment Considerations

Simple DB update for mob_pools.sql   should be handle by DB tool
